hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/genesis/GenesisUtil.java

```diff
@@ -68,6 +68,8 @@ private GenesisUtil() {
 
   private static final int DB_FILE_LEN = 7;
   private static final String TMP_DIR = "java.io.tmpdir";
+  private static final Random RANDOM = new Random();
+  private static final String SCM_CLIENT_DEFAULT_ADDRESS = "127.0.0.1:0";
```

Comment:
Given that this is also used for non-client SCM and OM addresses, wouldn't something like `RANDOM_LOCAL_ADDRESS` be a better name?

Reply:
Sure. Fixed it!